src/gui/torrentcreatordialog.h

```diff
@@ -60,6 +63,7 @@ private slots:
     void onAddFolderButtonClicked();
     void handleCreationFailure(const QString &msg);
     void handleCreationSuccess(const BitTorrent::TorrentCreatorResult &result);
+    void onPiecesCountCalculated(int count);
```

Comment:
Rename:
```suggestion
    void piecesCountCalculated(int count);
```

Reply:
@Chocobo1 
If you don't want it to be named in callback/event handler style, let it be named as regular function, i.e. `setPiecesCount()`.